digital_ingest_transformation_template.yaml

```diff
@@ -75,108 +57,126 @@ Resources:
             Version: '2012-10-17'
             Statement:
               - Effect: Allow
-                Action: 
+                Action:
                   - 'logs:CreateLogGroup'
                   - 'logs:CreateLogStream'
                   - 'logs:PutLogEvents'
-                Resource: 
-                  - !GetAtt AquariusLogGroup.Arn
+                Resource:
+                  - !GetAtt TransformationLogGroup.Arn
         - PolicyName: EcrPolicy
           PolicyDocument:
             Version: '2012-10-17'
             Statement:
               - Effect: Allow
-                Action: 
+                Action:
                   - 'ecr:BatchCheckLayerAvailability'
                   - 'ecr:GetDownloadUrlForLayer'
                   - 'ecr:BatchGetImage'
-                Resource: 
+                Resource:
                   - !Sub arn:aws:ecr:${AWS::Region}:${ContainerAccountId}:repository/${ApplicationName}
               - Effect: Allow
                 Action:
                   - 'ecr:GetAuthorizationToken'
                 Resource: '*'
 
-  AquariusUser:
+  TransformationUser:
     Type: AWS::IAM::User
+    UserName: TransformationUser
```

Comment:
Let's talk about this - I have typically not assigned usernames because names can cause problems with redeployments. See also the warning in the `UserName` property here: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-iam-user.html

Reply:
Per discussion, we can get rid of this because it's not adding anything to our use cases. The PhysicalID string isn't necessary.